.infrastructure/helm-chart/todoapp/charts/mysql/templates/statefulSet.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Chart.Name }}
+  namespace: {{ .Values.global.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+  replicas: {{ .Values.mysql.statefulSet }}
+  serviceName: {{ .Chart.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+    spec:
+      containers:
+      - name: {{ .Values.mysql.statefulSet.name }}
+        image: {{ .Values.mysql.statefulSet.repository }}:{{ .Values.mysql.statefulSet.version }}
+        env:
+        {{- range $k := .Values.mysql.secret }}
+        - name: {{ $k }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Chart.Name }}-secrets
+              key: {{ $k }}
+        {{- end }}
+          value: app_db
+        ports:
+        - name: mysql
+          containerPort: 3306
+        volumeMounts:
+        - name: data
+          mountPath: /var/lib/mysql
+          subPath: mysql
+        - name: config-map
+          mountPath: /docker-entrypoint-initdb.d
+        resources:
+          requests:
+            {{ .Values.mysql.statefulSet.resources.requests }}
+        livenessProbe:
+          {{ .Values.mysql.statefulSet.livenessProbe }}
+        readinessProbe:
+          {{ .Values.mysql.statefulSet.readinessProbe }}
```

Comment:
This is not changed.

Reply:
You mean I need to unite resources, levenessProbe and readinesP - and import by one link?

because I made changes from previous commit - ([screen](![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_helm_charts/assets/153445422/d6bf2e7e-a21a-4755-97fe-abdc5d343185)))

Reply:
Actually, resources, livenessProbe and readinesS on the same level in code line maybe will be better to keep them as 3 lines